### ISSUE TITLE
'check css' isn't showing the information associated with Parse Errors

### ISSUE THREAD

**Issue by dgmstuart** [nanoc/nanoc#484]
When running `nanoc check css` on https://github.com/dgmstuart/dgms_cv I get the following mystery errors:

```
Issues found!
  output/stylesheets/styles.css:
    [ ERROR ] css - Parse Error
    [ ERROR ] css - Parse Error
```

Running my output css through the [W3C CSS validator site](http://jigsaw.w3.org/css-validator/validator) I can see the source of the problem (apparently the validator currently chokes on the `@page` selector - used in print stylesheets):

    Parse Error @page { margin: 1.2cm; }
    Parse Error }

It would be nice if nanoc could display this information as well.

**Comment by ddfreyne** [comment]
Fixed by #504.

**Comment by dgmstuart** [comment]
Great :)
